To: vim_dev@googlegroups.com
Subject: Patch 7.3.408
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.408 (after 7.3.406)
Problem:    Missing declaration.
Solution:   Add the declaration. (John Marriott)
Files:	    src/gui_w48.c


*** ../vim-7.3.407/src/gui_w48.c	2012-01-20 17:57:47.000000000 +0100
--- src/gui_w48.c	2012-01-20 20:51:57.000000000 +0100
***************
*** 3286,3291 ****
--- 3286,3292 ----
  {
      char_u *tmp;
      int len;
+     WCHAR *res;
  
      tmp = convert_filter(s);
      if (tmp == NULL)
*** ../vim-7.3.407/src/version.c	2012-01-20 20:44:38.000000000 +0100
--- src/version.c	2012-01-20 20:52:56.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     408,
  /**/

-- 
BLACK KNIGHT: None shall pass.
ARTHUR:       I have no quarrel with you, brave Sir knight, but I must cross
              this bridge.
BLACK KNIGHT: Then you shall die.
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
